assets/swaggerV1.yaml

```diff
@@ -2042,12 +2042,29 @@ paths:
               id:
                 type: string
                 description: Must be a language supported by Grottocenter (see GET /languages route) using the 3-letters format.
+
+        - name: geogPolygon
+          in: query
+          required: true
+          description: Only the inside of the geometry attribute of a GeoJson
```

Comment:
A geographical polygon representation of the massif stored as a GeoJson using EPSG 4326

Reply:
Ta phrase `Only the inside of the geometry attribute of a GeoJson` n'est pas clair pour moi...Faut il passer ici du WKT et non du GeoJSON?

Reply:
![image](https://user-images.githubusercontent.com/74241345/164975064-4ce6a3b0-0102-4935-acf2-e4893e1bed6c.png)
Sur l'ensemble du geoJson, il ne faut passer que la proprieté `geometry` qui contient les coordonnées et le type

Reply:
A geographical polygon representation of the massif stored as the "geometry" attribute (only) of a GeoJson using EPSG 4326